python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -32,14 +32,14 @@
 from dagster._core.execution.context.invocation import BoundOpExecutionContext
 from dagster._core.pipes.client import PipesMessageReader
 
-ExtResult: TypeAlias = Union[MaterializeResult, AssetCheckResult]
+PipesResult: TypeAlias = Union[MaterializeResult, AssetCheckResult]
```

Comment:
this name feels weird to me. `PipesAssetResult` maybe?

Reply:
I want to keep this name generic to support the raw op use case. And there is no guarantee that we will _only_ have asset-based events going forward.

I'm going to land this as is and make a note of all names we need followups on.